mmv1/third_party/terraform/tests/resource_cloudfunctions_function_test.go.erb

```diff
@@ -918,3 +1002,152 @@ resource "google_cloudfunctions_function" "function" {
 }
 `, projectNumber, networkName, vpcConnectorName, vpcConnectorName, vpcIp, bucketName, zipFilePath, functionName, vpcConnectorName)
 }
+
+func testAccCloudFunctionsFunction_secretEnvVar(projectNumber, secretName, versionName, bucketName, functionName, versionNumber, zipFilePath string) string {
+	return fmt.Sprintf(`
+data "google_project" "project" {}
+
+resource "google_project_iam_member" "gcfadmin" {
```

Comment:
Why do we need to grant the auto generated service account permissions specifically here?

Reply:
must have been from another test setup. deleted it and test still works :)